RecoHI/HiTracking/python/HISelectedVertex_cfi.py

```diff
@@ -7,11 +7,57 @@
 )
 
 # select best of precise vertex, fast vertex, and beamspot
-hiSelectedVertex = cms.EDProducer("HIBestVertexProducer",
+hiSelectedPixelVertex = cms.EDProducer("HIBestVertexProducer",
     beamSpotLabel = cms.InputTag("offlineBeamSpot"),
     adaptiveVertexCollection = cms.InputTag("hiBestAdaptiveVertex"),
     medianVertexCollection = cms.InputTag("hiPixelMedianVertex")
 )
 
 # best vertex sequence
-bestHiVertex = cms.Sequence( hiBestAdaptiveVertex * hiSelectedVertex )
+bestHiVertex = cms.Sequence( hiBestAdaptiveVertex * hiSelectedPixelVertex ) # vertexing run BEFORE tracking
+
+from RecoHI.HiTracking.HIPixelAdaptiveVertex_cfi import *
+hiOfflinePrimaryVertices=hiPixelAdaptiveVertex.clone( # vertexing run AFTER tracking
+    TrackLabel = cms.InputTag("hiGeneralTracks"),
+                                       
+    TkFilterParameters = cms.PSet(
+        algorithm = cms.string('filterWithThreshold'),
+        maxNormalizedChi2 = cms.double(5.0),
+        minPixelLayersWithHits=cms.int32(3),    #0 missing pix hit
+        minSiliconLayersWithHits = cms.int32(5),#at least 8 hits total
+        maxD0Significance = cms.double(3.0),    #default 5.0, suppresses split vtxs
```

Comment:
The text in the comments appears to be out of sync from the values.
Please update or clarify.